src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.c

```diff
@@ -186,34 +187,34 @@ void func_80A6DC7C(EnHorseZelda* this) {
     Animation_PlayOnce(&this->skin.skelAnime, sAnimationHeaders[this->animationIndex]);
 }
 
-void func_80A6DCCC(EnHorseZelda* this, PlayState* play) {
+void EnHorseZelda_Stop(EnHorseZelda* this, PlayState* play) {
     this->actor.speedXZ = 0.0f;
     if (SkelAnime_Update(&this->skin.skelAnime)) {
-        func_80A6DC7C(this);
+        EnHorseZelda_ResetAnimation(this);
     }
 }
 
-void func_80A6DD14(EnHorseZelda* this) {
-    f32 sp34;
+void EnHorseZelda_Gallop(EnHorseZelda* this) {
```

Comment:
this sets `action=1` and sets things up for the gallop animation, so this is more of an action setup function
The actual action would be the "Go" function

So I'd suggest
`EnHorseZelda_Gallop` -> `EnHorseZelda_SetupGallop`
`EnHorseZelda_Go` -> `EnHorseZelda_Gallop`